generate-config.js

```diff
@@ -704,7 +704,7 @@ const formBrowserWorkflow = (browserImages) => {
 const formIncludedWorkflow = (images) => {
   // skip images that have been built already
   const isSkipped = (tag) => {
-    return semver.lt(tag, '6.0.0')
+    return semver.lt(tag, '6.0.1')
```

Comment:
we don't have to bump this - this is just internal to avoid generating lots of CI jobs in circle.yml for Docker images that are already built. So skipping all images before v6 is ok to keep for a long time